test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1146,6 +1146,100 @@ public void PcaOnnxConversionTest()
             Done();
         }
 
+        private class HashData
+        {
+            public uint Data { get; set; }
+        }
+
+        [Fact]
+        public void MurmurHashKeyTest()
```

Comment:
>MurmurHashKeyTest [](start = 20, length = 17)

If ONNX can handle vector columns, please also add a test for that.

Reply:
Added!

Reply:
Is it possible to have a scalar and a vector test for each type? Also, if you bring back support for ordered hashing in the non-vector case, to test both ordered and non ordered for scalars?

---
In reply to: [414106161](https://github.com/dotnet/machinelearning/pull/5013#discussion_r414106161) [](ancestors = 414106161)

Reply:
done!